de.metas.util/src/main/java/de/metas/util/rest/ExternalId.java

```diff
@@ -63,12 +63,4 @@ public static String toValue(@Nullable final ExternalId externalId)
 		}
 		return externalId.getValue();
 	}
-	
```

Comment:
does the json we need to send to the API still look the same, even without this getter?

Reply:
yes.